native/c/test/zutils.cpp

```diff
@@ -65,9 +69,23 @@ string get_random_string(const int length, const bool allNumbers)
     seeded = true;
   }
   string ret = "";
+  // A simple array to map an index (0-25) to an EBCDIC uppercase character code
+  const unsigned char EBCDIC_UPPERCASE_LETTERS[] = {
+      0xC1, 0xC2, 0xC3, 0xC4, 0xC5, 0xC6, 0xC7, 0xC8, 0xC9, // A-I
+      0xD1, 0xD2, 0xD3, 0xD4, 0xD5, 0xD6, 0xD7, 0xD8, 0xD9, // J-R
+      0xE2, 0xE3, 0xE4, 0xE5, 0xE6, 0xE7, 0xE8, 0xE9        // S-Z
+  };
```

Comment:
- (nitpick) A-Z are contiguous in the EBCDIC character set - thoughts on using addition with EBCDIC A's hex value to reduce stack usage?

```suggestion
  const unsigned char EBCDIC_A = 0xC1;
```

Suggestion below shows an example 😋 

- This function declares a new const array on the stack w/ 26 elements every time its used (assuming no/minimal optimization in tests), if you'd prefer to keep this consider moving it outside of the function or marking it as static